src/main/kotlin/no/nav/klage/oppgave/api/OppgaveController.kt

```diff
@@ -89,13 +89,16 @@ class OppgaveController(
         return ResponseEntity.noContent().location(uri).build()
     }
 
-    @GetMapping("/ansatte/{navIdent}/oppgaver/{id}")
-    fun getOppgave(
-        @PathVariable navIdent: String,
-        @PathVariable("id") oppgaveId: String
-    ): Oppgave {
-        logger.debug("getOppgave is requested: {}", oppgaveId)
-        return oppgaveFacade.getOppgave(oppgaveId.toLongOrException())
+    @GetMapping("/klagebehandlinger/{id}")
+    fun getKlagebehandling(
+        @PathVariable("id") klagebehandlingId: String
```

Comment:
Det jeg er mest spent på her er kanskje flyten i applikasjonen.. Hvor får vi klagebehandlingId fra her? Hvordan passer dette inn i GUI-flyten? Trengs det f.eks en "getAllKlagebehandlingForSaksbehandler" metode? Eller trengs det et oppslag av klagebehandling fra oppgaveId i stedet for/i tillegg til fra klagebehandlingId?

Reply:
For nå så sendes oppgaveId inn i stedet.